From f429fc1a072d4bb35e622a1012a5a52914eba4e3 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Thu, 19 Oct 2023 10:34:58 +0100
Subject: [PATCH] drm/connector: Change DRM card alias from underscore to
 hyphen

Apparently aliases are only allowed lower case and hyphens,
so swap the use of underscore to hyphen.

Fixes: 3aa1f2477545 ("drm: Look for an alias for the displays to use as the DRM device name")
Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 drivers/gpu/drm/drm_connector.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/gpu/drm/drm_connector.c
+++ b/drivers/gpu/drm/drm_connector.c
@@ -112,7 +112,7 @@ static struct drm_conn_prop_enum_list dr
 };
 
 #define MAX_DT_NODE_NAME_LEN	20
-#define DT_DRM_NODE_PREFIX	"drm_"
+#define DT_DRM_NODE_PREFIX	"drm-"
 
 static void drm_connector_get_of_name(int type, char *node_name, int length)
 {
